### PR TITLE
add folder structure for navbar and footer

### PR DESCRIPTION
**Pull request by BIKRAM-SAHA**
Solves issue: #2 
changes made:
added files Navbar.tsx, Navbar.module.scss, Navbar.types.ts
added files Footer.tsx, Footer.module.scss, Footer.types.ts

### CODE REVIEW

File: components/commons/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { FooterProps } from "./Footer.types";
+import "./styles/Footer.module.scss";
+
+export default function Footer({}: FooterProps) {
```
Comment (FaizBShah) [inline components/commons/footer/Footer.tsx]: `FC` stands for FunctionalComponent Type, which is a type provided by React for Functional Components

```suggestion
const Footer: FC<Props> = ({}) => {
```

File: components/commons/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { FooterProps } from "./Footer.types";
```
Comment (FaizBShah) [inline components/commons/footer/Footer.tsx]: import `FC` from `react` in the code

File: components/commons/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+import "./styles/Footer.module.scss";
+
+export default function Footer({}: FooterProps) {
+  return <div>Footer here</div>;
+}
```
Comment (FaizBShah) [inline components/commons/footer/Footer.tsx]: Add `export default Footer;` at the end

File: components/commons/footer/Footer.tsx (hunk 2 above)
Comment (FaizBShah) [inline components/commons/footer/Footer.tsx]:
```suggestion
import { FooterProps as Props } from "./Footer.types";
```

File: components/commons/navbar/Navbar.tsx
Comment (FaizBShah) [inline components/commons/navbar/Navbar.tsx:1]: Same changes for `Navbar.tsx` file as `Footer.tsx`

File: components/commons/footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { FooterProps } from "./Footer.types";
+import "./styles/Footer.module.scss";
```
Comment (FaizBShah) [inline components/commons/footer/Footer.tsx]:
```suggestion
import styles from "./styles/Footer.module.scss";
```

**Review by FaizBShah — CHANGES_REQUESTED** [review]

**Comment by BIKRAM-SAHA** [comment]
Done the changes in the new commit. Can you please review again?

**Review by FaizBShah — APPROVED** [review]
LGTM

**Pull request merged**